2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -94,61 +94,34 @@ Per la comunicazione tra le sue componenti, con l'utente e con servizi esterni,
 
 Non sono definiti vincoli o limiti sulle memorie primaria e secondaria.
 
-==== Operazioni
-
-//non ho molto chiaro cosa si richiedesse in questa sezione, sotto riporto lo standard
-//Specify the normal and special operations required by the user such as:
-// - the various modes of operations in the user organization*/
-Il sistema permette di attuare e visualizzare diverse operazioni di gestione del magazzino, in modo da semplificare l'interazione tra gestori e addetti.
-Il sistema deve permettere al *responsabile del magazzino* le seguenti operazioni:
-- creazione del magazzino (manualmente o tramite caricamento di un file SVG);
-- modificare le dimensioni dell'ambiente 3D (lunghezza, larghezza, altezza);
-- creare e modificare le dimensione di scaffali (larghezza, profondità e altezza);
-- eliminare gli scaffali;
-- riposizionare gli scaffali nell'ambiente;
-- definire il numero di piani dello scaffale;
-- definire il numero di bin per piano;
-- creare e modificare i bin (lunghezza, larghezza e altezza);
-- leggere informazioni riguardo agli scaffali e al contenuto dei bin;
-- eliminare i bin;
-- caricare i dati da database;
-- richiedere lo spostamento di un prodotto da un bin a un altro;
-- ricercare un prodotto nel magazzino;
-- visualizzare ed esplorare il magazzino.
-
-Modifiche alla posizione dei prodotti comporta l'invio di una notifica a gli *addetti* che provvederanno ad attuare lo spostamento.
-
 ==== Requisiti di adattamento al contesto
-//The site adaptation requirements include:
-//a) definition of the requirements for any data or initialization sequences that are specific to a given
-//site, mission or operational mode (e.g., grid values, safety limits, etc.);
-//b) specification of the site or mission-related features that should be modified to adapt the software
-//to a particular installation.
-WMS3 non richiede l'installazione di particolari librerie software o specifici sistemi operativi grazie a Docker, che permette di costruire dei container che si occupano di gestire le dipendenze e Docker Compose che si occupa di gestire applicazioni multi-container.\
-Il sistema permette libertà anche dal punto di vista del sistema di notifiche interne al magazzino e del database, visto che l'applicazione permette configurazione tramite API RESTful.
+
+WMS3 per essere eseguito richiede:
+- un *browser* che supporta WebGL 2.0 (per le specifiche riguardanti i vari browser compatibili consultare la sezione @vincoli);
+- *Node.js* versione 20.11.0 (latest LTS) o superiore;
+- Un database relazionale che si interfacci con le API fornite dal gruppo (il gruppo utilizza *Postgresql* versione 16.1);
+- *Docker Compose* versione 2.23.3 o superiore;
+- *Docker* versione 24.0.7 o superiore;
+
+Il gruppo ha deciso di utilizzare la tecnologia Docker per permettere una maggiore portabilità e facilitare il deploy, mentre Docker Compose serve a gestire le applicazioni multi-container. \
```

Comment:
Metterei un bel punto dopo "deploy". La frase successiva la riscriverei in "La gestione di più container simultanei avviene mediante Docker Compose."